drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -274,15 +274,32 @@ RasterizerGLES3::~RasterizerGLES3() {
 void RasterizerGLES3::prepare_for_blitting_render_targets() {
 }
 
-void RasterizerGLES3::_blit_render_target_to_screen(RID p_render_target, DisplayServer::WindowID p_screen, const Rect2 &p_screen_rect) {
+void RasterizerGLES3::_blit_render_target_to_screen(RID p_render_target, DisplayServer::WindowID p_screen, const Rect2 &p_screen_rect, uint32_t p_layer) {
 	GLES3::RenderTarget *rt = GLES3::TextureStorage::get_singleton()->get_render_target(p_render_target);
+
 	ERR_FAIL_COND(!rt);
 
-	glBindFramebuffer(GL_READ_FRAMEBUFFER, rt->fbo);
+	GLuint read_fbo = 0;
+	if (rt->view_count > 1) {
+		glGenFramebuffers(1, &read_fbo);
```

Comment:
Do we want to generate a new FBO every frame or should we cache these? @clayjohn is there any performance concern here?

Reply:
We certainly could cache them! I'd imagine putting like `rt->layer_fbos = new GLint[rt->view_count];` on the render target - we could create them on demand, and clear them out when the render target gets updated.

But I don't really know if creating/destroying these framebuffers here is a performance issue that makes it worth doing that? I'd be happy to implement it if so :-)

Reply:
It doesn't really matter one way or another. In theory caching could be slightly faster, but in practice it really makes no difference because the cost of creating and binding the framebuffer is essentially the same as just the cost of binding the framebuffer.